### ISSUE TITLE
SA1008 (OpeningParenthesisMustBeSpacedCorrectly) is reported in for loop

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#876]
The following code will report SA1008
```csharp
for (var i = leadingTrivia.Count - 1; (i >= 0) && leadingTrivia[i].IsKind(SyntaxKind.WhitespaceTrivia); i--)
```

**Comment by sharwell** [comment]
Blocked on #193.
